test/parallel/test-util-inspect.js

```diff
@@ -82,7 +82,8 @@ assert.strictEqual(util.inspect(Object.assign(new String('hello'),
                    { [Symbol('foo')]: 123 }), { showHidden: true }),
                    '{ [String: \'hello\'] [length]: 5, [Symbol(foo)]: 123 }');
 
-assert.strictEqual(util.inspect(process.stdin._handle._externalStream),
+const JSStream = process.binding('js_stream').JSStream;
```

Comment:
Destructuring can be used here, and generally these imports are positioned at the top of the file.